### PR TITLE
User Documentation for Skeleton Tool

### PR DESCRIPTION
**Pull request by Aa20475**
Please verify and suggest any changes!

### CODE REVIEW

File: source/tools/skeleton.rst [hunk 1]
```diff
@@ -0,0 +1,82 @@
+.. _tools_skeleton:
+
+########################
+    Skeleton Tool
+########################
+
+
+Introduction
+====
```
Comment (morevnaproject) [inline source/tools/skeleton.rst:9]: Got a warning here:
```
WARNING: Title underline too short.
```

File: source/tools/skeleton.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Synfig also supports animating objects using bones and the Skeleton tool makes it easy to build skeletons of Bones.
+
+Usage
+====
```
Comment (morevnaproject) [inline source/tools/skeleton.rst:14]: Got a warning here:
```
WARNING: Title underline too short.
```

File: source/tools/skeleton.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    :alt: Make parent to active bone
+
+*When you are modifying a skeleton deformation layer using the Skeleton tool, it is to be noted that-*
+
+* The layer gets disabled
+* Width handles are turned on.
+* Any changes you make to the rest position effects the deformation too.
+
+Options
+====
```
Comment (morevnaproject) [inline source/tools/skeleton.rst:72]: Got a warning here:
```
WARNING: Title underline too short.
```

File: source/tools/skeleton.rst [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+If you try to use this tool when the selected layer is not any of the bone layers, a new layer is created with the bone added. The layer that is to be created can be set in the tool options menu. You can always create an empty bone layer using the "Create Layer" button in the tool options menu.
+
+.. image:: ./skeleton_dat/drag_bone_toggle.gif
+    :width: 600px
+    :align: center
+    :alt: Click-Drag-Release bone with create layer
+
+The parent child relationships of bones are represented by a dashed line from parent's tip to the child's origin. 
+
+When you click on a bone, the bone is set as the **"Active Bone"**. Active bone is highlighted with a yellow outline. Any bone that's created is added as a child to the active bone. When there is no active bone set, then the bones created after will have no parent bone.
```
Comment (morevnaproject) [inline source/tools/skeleton.rst:44]: When there is no active bone set, then the **bone** created after will have **no parent.**

File: source/tools/skeleton.rst [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+.. image:: ./skeleton_dat/active_bone.gif
+    :width: 600px
+    :align: center
+    :alt: Active Bone
+
+The handles of the bone will have same meaning as they had before.A point to be noted is that, you can even click on a Handle to set a bone as "Active Bone". When you use a handle to modify a bone, note that the bone is set as the "Active Bone".
```
Comment (morevnaproject) [inline source/tools/skeleton.rst:51]: Need to add space before "A point to be noted". ^__^

File: source/tools/skeleton.rst [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+Select "Skeleton Tool" in the Toolbox Panel or use the keyboard shortcut <Alt><o> to enable it. To leave the tool, you can just select another tool from the toolbox.
+
+.. image:: ./skeleton_dat/select_tool.gif
+    :width: 600px
+    :align: center
+    :alt: Select tool
+
+There are two types of layers in Synfig that are built with bones: Skeleton Layer and Skeleton deformation layer. We can build any of these layers.
+
+You can create a bone by either-
```
Comment (morevnaproject) [inline source/tools/skeleton.rst:25]: I think "-" symbol at the end should be removed. ^__^

File: source/tools/skeleton.rst [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    :alt: Handle drag to set active bone
+
+The parent-child relationships of bones can be changed by using the **"Make parent to active bone"** option in the context menu of any bone handle.
+
+.. image:: ./skeleton_dat/make_parent.gif
+    :width: 600px
+    :align: center
+    :alt: Make parent to active bone
+
+*When you are modifying a skeleton deformation layer using the Skeleton tool, it is to be noted that-*
```
Comment (morevnaproject) [inline source/tools/skeleton.rst:65]: I think there is no need to wrap this sentence into "*".

Also, I suggest to remove "-" at the end.

File: source/tools/skeleton.rst [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+* The layer gets disabled
```
Comment (morevnaproject) [inline source/tools/skeleton.rst:67]: Add "." at the end.

File: source/tools/skeleton.rst [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+* Width handles are turned on.
+* Any changes you make to the rest position effects the deformation too.
+
+Options
+====
+
+You can find the following options in the tool options menu of this tool-
```
Comment (morevnaproject) [inline source/tools/skeleton.rst:74]: "You can find the following options in the tool options menu:"

**Review by morevnaproject — CHANGES_REQUESTED** [review]
Please check review comments.

File: source/tools/skeleton.rst (hunk 9 above)
Comment (morevnaproject) [reply source/tools/skeleton.rst]: @Aa20475  Please add ":" at the end ^__^

File: source/tools/skeleton.rst (hunk 8 above)
Comment (morevnaproject) [reply source/tools/skeleton.rst:67]: @Aa20475 This one still remains. ^__^

File: source/tools/skeleton.rst (hunk 9 above)
Comment (Aa20475) [reply source/tools/skeleton.rst]: Now?

File: source/tools/skeleton.rst (hunk 8 above)
Comment (Aa20475) [reply source/tools/skeleton.rst]: Now?

**Pull request merged**